tests/unit/components/mixins/Messages.spec.ts

```diff
@@ -0,0 +1,51 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+import Messages from '@/components/mixins/Messages';
+import { NS_USER, NS_MESSAGES } from '@/store/namespaces';
+
+Vue.use( Vuex );
+
+function mockStore( getMessage: ( inLanguage: string, messageKey: string ) => string | null ) {
```

Comment:
The parameter should be the value returned by the getter, not the getter itself.